src/main/java/dev/dirs/UserDirectories.java

```diff
@@ -327,6 +327,18 @@ private UserDirectories() {
         templateDir   = null;
         videoDir      = homeDir + "/Movies";
         break;
+      case IBMI:
+        homeDir       = System.getProperty("user.home");
+        audioDir      = homeDir + "/Music";
+        desktopDir    = homeDir + "/Desktop";
+        documentDir   = homeDir + "/Documents";
+        downloadDir   = homeDir + "/Downloads";
+        fontDir       = defaultIfNullOrEmptyExtended(System.getenv("XDG_DATA_HOME"), "/fonts",  homeDir, "/.local/share/fonts");
+        pictureDir    = homeDir + "/Pictures";
+        publicDir     = homeDir + "/Public";
+        templateDir   = null;
+        videoDir      = homeDir + "/Movies";
+        break;
```

Comment:
Is there some specification for these directories on IBM iSeries I can read up on?

Reply:
Unfortunately not, since we (IBM) haven't really delivered any window managers where these values make sense. But when we do, these are the conventions we plan to follow. 

If you'd prefer, we can null them out and circle back when we deliver UI where these special dirs matter. 

Thanks!

Reply:
Thanks! Are these values derived from the macOS/Linux ones? (It's equal to macOS except for fonts/equal to XDG user dirs on Linux, except Videos/Movies.)

Reply:
Yep, exactly, we predict it'll be a blend of the macOS and Linux conventions.

Reply:
Ok, thanks, understood!